data/test_assets/dataset_tests/lights/dataset_test_lights.lighting_config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "lights":{
+        "0": { "position": [2.45833,0.1,3.84271], "intensity": 1.4, "color": [1,1,0.95], "type": "point"},
+        "1": { "position": [3.28035,-0.1,8.15357], "intensity": 1.1, "color": [1,1,0.95], "type": "point"},
+        "2": { "position": [0.35,0.5,3.5], "intensity": 1.3, "color": [0.93,0.98,1], "type": "point"},
+        "3": { "position": [0.35,0.5,1.0], "intensity": 1.3, "color": [0.93,0.98,1], "type": "point"},
+        "4": { "direction": [0.0,-1.0,0.0], "intensity": 1.3, "color": [0.93,0.98,1], "type": "area"},
+        "5": { "position": [-5.59517,5,-8.33407], "intensity": 120, "color": [1,1,1], "type": "point"},
+        "6": { "position": [8.65492,1.7,3.23571], "intensity": 15, "color": [1,1,1], "type": "point"},
+        "7": { "position": [5,5,0], "intensity": -10, "color": [1,1,1], "type": "point"},
+        "8": { "direction": [1.80922,-1.0,0.5], "intensity": -0.4, "color": [1,1,1], "type": "area"},
```

Comment:
I don't think we support area lights. And wouldn't an area light need to specify the area somehow?

Also what happened to directional lights?

Reply:
Thanks Eric.  That's a typo, should be directional.

Reply:
@jturner65 Do not have any validators to catch these errors?

Reply:
Not yet, not in this PR.  This is just the initial commit of the testing data used to verify the functionality of the dataset and scene import.  JSON validators would be great, however.

Reply:
@jturner65 Done: https://github.com/facebookresearch/habitat-sim/pull/854

Reply:
Just FYI, in this case, the issue was incorrect data value for "type", not any invalid json configurations.   The added validator does not check for values, does it?